daprdocs/content/en/getting-started/install-dapr-selfhost.md

```diff
@@ -52,8 +52,8 @@ dapr --version
 
 Output should look like this:
 ```
-CLI version: 1.0.0
-Runtime version: 1.0.0
+CLI version: 1.0.1
```

Comment:
was a new version released?
this should still be 1.0.0

Reply:
Find replace error. Fixed. Thanks.